helm/kadalu/charts/csi-nodeplugin/templates/daemonset.yaml

```diff
@@ -63,6 +63,8 @@ spec:
               value: {{ .Values.verbose | quote }}
             - name: CSI_ROLE
               value: "nodeplugin"
+            - name: OVERRIDE_METHOD
```

Comment:
Please rename this to something that describes the actual use; eg 
- USE_VMEXEC_MOUNTS

etc